src/practices/LanguageIndependent/DoesPullRequests.ts

```diff
@@ -31,6 +32,21 @@ export class DoesPullRequestsPractice implements IPractice {
       filter: { state: PullRequestState.all },
     });
     const repoCommits = await ctx.collaborationInspector.listRepoCommits(ownerAndRepoName.owner, ownerAndRepoName.repoName);
+    this.setData(
+      pullRequests.items.map((pr) => {
+        return {
+          id: pr.id,
+          name: pr.title, //TODO send pr url too?
```

Comment:
Yes, send also url, so we can use it in the dashboard so you can get to the PR directly from dashboard.